start-mycroft.sh

```diff
@@ -131,39 +135,39 @@ function launch-background() {
     fi
 
     # Security warning/reminder for the user
-    if [[ "${1}" == "bus" ]] ; then
+    if [ "${1}" = "bus" ] ; then
         echo "CAUTION: The Mycroft bus is an open websocket with no built-in security"
         echo "         measures.  You are responsible for protecting the local port"
         echo "         8181 with a firewall as appropriate."
     fi
 
     # Launch process in background, sending logs to standard location
-    python3 -m ${_module} $_params >> /var/log/mycroft/${1}.log 2>&1 &
+    python3 -m ${_module} "$_params" >> /var/log/mycroft/"${1}".log 2>&1 &
 }
 
-function launch-all() {
+launch_all() {
     echo "Starting all mycroft-core services"
-    launch-background bus
-    launch-background skills
-    launch-background audio
-    launch-background voice
-    launch-background enclosure
+    launch_background bus
+    launch_background skills
+    launch_background audio
+    launch_background voice
+    launch_background enclosure
 }
 
-function check-dependencies() {
+check_dependencies() {
     if [ -f .dev_opts.json ] ; then
         auto_update=$( jq -r ".auto_update" < .dev_opts.json 2> /dev/null)
     else
         auto_update="false"
     fi
-    if [ "$auto_update" == "true" ] ; then
+    if [ "$auto_update" = "true" ] ; then
         # Check github repo for updates (e.g. a new release)
         git pull
     fi
 
-    if [ ! -f .installed ] || ! md5sum -c &> /dev/null < .installed ; then
+    if [ ! -f .installed ] || ! md5sum -c 2>&1 > /dev/null < .installed ; then
```

Comment:
what's the reason to also shove `stderr` to `/dev/null`?

Reply:
No reason really, that's just a habit of mine I guess. Good call, I'll change it.